README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center"><a href="https://newpipe.net"><img src="assets/new_pipe_icon_5.png" width="150"></a></p> 
 <h2 align="center"><b>NewPipe</b></h2>
-<h4 align="center">A libre lightweight streaming frontend for Android.</h4>
+<h4 align="center">A libre lightweight streaming front-end for Android.</h4>
```

Comment:
Both the original and your suggestion are correct, though I prefer the one with a hyphen.